### PR TITLE
Fix/dotnet code snippets for text and goto

### PR DESCRIPTION
**Pull request by Honyii**
This is a dotnet docs update for Page.GetByText and age.GotoAsync

### CODE REVIEW

**Comment by Skn0tt** [comment]
I think the snippets are correct as they are. Could you outline what's the problem in your opinion?

**Comment by Honyii** [comment]
For dotnet, it should be Page and not page and this is visible in almost
all the dotnet snippets in the doc.

**Comment by Honyii** [comment]
Also, my plan is to have a look at the snippets used for dotnet and update
as much as possible.

**Comment by Skn0tt** [comment]
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions says that local variables should be in `camelCase`, and the snippets this PR updates are all assuming `page` to be a local variable. Am I missing something?

**Comment by Honyii** [comment]
Yes, for local variables that's why some of the snippets were not updated
because a variable creation was clearly stated in the snippets. For example;
https://playwright.dev/dotnet/docs/auth
https://playwright.dev/dotnet/docs/network
However, have a look at the different pattern for nodesjs, python, java,
dotnet in this snippet
https://playwright.dev/docs/api/class-page#page-get-by-text

So for an absolute beginner that wants to get started, snippets are guides
and they are mostly used the way it is written in the docs- using e.g
page.GetByText for a dotnet app will fail.

**Comment by Honyii** [comment]
Or isn’t this the expectation for the dotnet docs?

**Review by mxschmitt — COMMENTED** [review]
Hello! Please revert the changes inside `docs/src/api` folder since they target the library where page is usually lowercase and the rest are guides which target Microsoft.Playwright.NUnit etc.

Also now a lot of methods use now mixed casing, some a bit of `Page` and some a bit of `page`. e.g. looking at input.md.

**Comment by Honyii** [comment]
Alright, thanks for the explanation

File: docs/src/input.md [hunk 1]
```diff
@@ -301,23 +301,23 @@ page.get_by_text("Item").click(position={ "x": 0, "y": 0})
 await page.GetByRole(AriaRole.Button).ClickAsync();
```
Comment (pavelfeldman) [inline docs/src/input.md:301]:
```suggestion
await Page.GetByRole(AriaRole.Button).ClickAsync();
```

File: docs/src/input.md [hunk 2]
```diff
@@ -460,7 +460,7 @@ page.get_by_role("textbox").press("$")
 
 ```csharp
 // Hit Enter
-await page.GetByText("Submit").PressAsync("Enter");
+await Page.GetByText("Submit").PressAsync("Enter");
 
 // Dispatch Control+Right
 await page.GetByRole(AriaRole.Textbox).PressAsync("Control+ArrowRight");
```
Comment (pavelfeldman) [inline docs/src/input.md:466]:
```suggestion
await Page.GetByRole(AriaRole.Textbox).PressAsync("Control+ArrowRight");
```

File: docs/src/locators.md [hunk 3]
```diff
@@ -1265,7 +1265,7 @@ new_email.click()
 
 ```csharp
 var newEmail = page.GetByRole(AriaRole.Button, new() { Name = "New" });
```
Comment (pavelfeldman) [inline docs/src/locators.md:1267]:
```suggestion
var newEmail = Page.GetByRole(AriaRole.Button, new() { Name = "New" });
```

File: docs/src/mock.md [hunk 4]
```diff
@@ -376,7 +376,7 @@ await page.RouteFromHARAsync("./hars/fruit.har", new ()
 );
 
 // Go to the page
-await page.GotoAsync("https://demo.playwright.dev/api-mocking");
+await Page.GotoAsync("https://demo.playwright.dev/api-mocking");
 
 // Assert that the Playwright fruit is visible
 await page.ExpectByTextAsync("Playwright", new() { Exact = true }).ToBeVisibleAsync();
```
Comment (pavelfeldman) [inline docs/src/mock.md:382]:
```suggestion
await Page.ExpectByTextAsync("Playwright", new() { Exact = true }).ToBeVisibleAsync();
```

File: docs/src/network.md [hunk 5]
```diff
@@ -338,7 +338,7 @@ response = response_info.value
 ```csharp
 // Use a glob URL pattern
 var waitForResponseTask = page.WaitForResponseAsync("**/api/fetch_data");
```
Comment (pavelfeldman) [inline docs/src/network.md:340]:
```suggestion
var waitForResponseTask = Page.WaitForResponseAsync("**/api/fetch_data");
```

**Review by pavelfeldman — COMMENTED** [review]
I marked some more, but it does not look like this effort is systematically making things more consistent, let me close the PR.

**Pull request closed without merging**